portfolio/src/main/webapp/infiniterun/gamepage.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Infinite Run</title>
+        <link rel="stylesheet" href="gamestyle.css">
+        <script src="gamescript.js"></script>
+    </head>
+    <body>
+        <div id="content">
+            <div style="text-align: center; border: 1px solid">
```

Comment:
nit: put all styles in CSS files
https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

You can probably create a class="content" or something and save the styling. Seems like this is repeated across your pages.